### PR TITLE
Added capability to set task tags, modified message on browser start (closes #9, closes #10)

### LINKED ISSUE #9: Add the capability to set next task tags: build, tags

### LINKED ISSUE #10: Modify message on browser start.
Change from
'Browser stardet. See ...'
to
'chrome started. See ...'

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin
PR

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -44,7 +49,7 @@ export default class SaucelabsConnector {
         return JSON.parse(response.body).concurrency[this.username].remaining.overall;
     }
 
-    async startBrowser (browser, url, jobName, timeout = null) {
+    async startBrowser (browser, url, timeout = null) {
```
Comment (AlexanderMoskovkin) [inline src/index.js]: We should can an ability to run tasks with different jobNames (builds or tags) with the same saucelabsConnector instance. So, this args relates to the `startBrowser` fn. Let's move them from the constructor to this function.
They can be looked like this:

``` js
async startBrowser (browser, url, timeout = null, { jobName, build, tags })
```

And they should be optional

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "saucelabs-connector",
-  "version": "0.0.2",
+  "version": "0.0.3",
```
Comment (AlexanderMoskovkin) [inline package.json]: We change public API with this PR, so we should change the minor version (`0.1.0`)
FYI: `version: {MAJOR}.{MINOR}.{PATCH}` (http://semver.org/)

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**